### DISCUSSION TITLE
Regarding gsoc project in deeplearning

### DISCUSSION THREAD

**Discussion by Tan141101** [alan-turing-institute/sktime#4213] [Ideas]
Hello, 
I would like to suggest that with building deep learning modules for forecasting we could also use some examples as well like pet projects or some implementation of a research paper to help users also learn while using the sktime framework also would love to contribute in this project if you see me fit.

**Comment by fkiraly** [comment]
Welcome to `sktime`!

For getting started, have a look at our "getting started" guide:
https://github.com/sktime/sktime/issues/1147
Also feel free to join any of the community meet-ups on Fridays (discord)

Deep learning for forecasting is indeed one of the projects we have in mind for GSoC 2023!
https://github.com/sktime/mentoring/blob/main/internships/projects_2023.md
